chore(docs): Final Docs Migration Cleanup (#1431)
hashicorp/terraform-cdk: website/docs/cdktf/release/index.mdx

---
page_title: Releases
description: >-
  We release CDK for Terraform regularly. Find a link to the changelog and upgrade guides. Use feature flags to enable new behaviors.
---

# Releases

-> **Note:** CDK for Terraform is currently in [beta](#project-maturity-and-production-readiness).

> We release CDK for Terraform (CDKTF) regularly. The [CHANGELOG on github](https://github.com/hashicorp/terraform-cdk/blob/main/CHANGELOG.md) contains details about bug fixes, new features, and deprecations.

Reviewer: ```suggestion
We release CDK for Terraform (CDKTF) regularly. The [CHANGELOG on Github](https://github.com/hashicorp/terraform-cdk/blob/main/CHANGELOG.md) contains details about bug fixes, new features, and deprecations.
```

---
page_title: Releases
description: >-
  We ship a CDK for Terraform release each month. Find a list of recent releases and a link to our changelog.
---

# Releases

-> **Note:** CDK for Terraform is currently in [beta](#project-maturity-and-production-readiness).

> We ship a new Terraform Enterprise release each month. Recent releases are listed below. Our [CHANGELOG on github](https://github.com/hashicorp/terraform-cdk/blob/main/CHANGELOG.md) contains details about bug fixes, new features, and deprecations.

Reviewer 2: ```suggestion
We ship CDK for Terraform releases regularly. Recent releases are listed below. Our [CHANGELOG on github](https://github.com/hashicorp/terraform-cdk/blob/main/CHANGELOG.md) contains details about bug fixes, new features, and deprecations.
```

- [release name](link to changelog)

## Upgrade Guides

> - [Upgrading to v0.7](/cdktf/release/upgrade-guide-v0-7)

Reviewer 2: So we're going to move these from the GitHub repo to standalone docs pages?

Author: yes! I will add a line about this - great idea!
